tests/unit/commands/_utils/test_options.py

```diff
@@ -527,3 +528,20 @@ def test_skip_without_hook_package(self):
             skip_prepare_infra_callback(ctx_mock, param_mock, True)
 
         self.assertEqual(str(ex.exception), "Missing option --hook-name")
+
+
+class TestNextCommandSuggestions(TestCase):
+    def test_generate_next_command_recommendation(self):
+        listOfTuples = [("command1", "description1"), ("command2", "description2"), ("command3", "description3")]
+        output = generate_next_command_recommendation(listOfTuples)
+        expectedOutput = """
+Commands you can use next:
+================================
+
+[*] command1: description1
+[*] command2: description2
+[*] command3: description3
```

Comment:
The example in the summary has it the other way around.

Reply:
will fix!